x-pack/plugins/security_solution/server/lib/detection_engine/signals/executors/eql.ts

```diff
@@ -60,89 +62,93 @@ export const eqlExecutor = async ({
   wrapHits: WrapHits;
   wrapSequences: WrapSequences;
 }): Promise<SearchAfterAndBulkCreateReturnType> => {
-  const result = createSearchAfterReturnType();
-
   const ruleParams = completeRule.ruleParams;
+  agent.addLabels({
+    security_solution_rule_id: ruleParams.ruleId,
```

Comment:
We can remove this label (and other `security_solution_rule_id` labels) after https://github.com/elastic/kibana/pull/117504 is merged.